build/mac/build_delta_installer.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import optparse
+import subprocess
+import sys
+
+
+def Main(argv):
+  parser = optparse.OptionParser('%prog [options]')
+  parser.add_option('--binary-delta', dest='binary_delta_path', action='store',
+      type='string', default=None, help='The path of BinaryDelta binary.')
+  parser.add_option('--sign-update', dest='sign_update_path', action='store',
+      type='string', default=None, help='The path of sign_update binary')
+  parser.add_option('--old-app', dest='old_app_path', action='store',
```

Comment:
let's use `current` and `previous` vs `new` and `old`

Reply:
in fact it might not be the current/latest version so it makes sense how it is now

Reply:
So, it's ok to use `old` and `new`?

Reply:
yes, we can keep